install/kubernetes/helm/istio/charts/mixer/values.yaml

```diff
@@ -1,13 +1,19 @@
 replicaCount: 1
 image:
-  repository: docker.io/istio/pilot
+  repository: docker.io/istio/mixer
   tag: 0.4.0
```

Comment:
I realize you may have changed all the tags to 0.4.0, although 0.5.0 might be more appropriate as the 0.5.0 release is coming shortly.

Reply:
yes, I need to update `istio.Version` and generate the helm values using the build script

Reply:
cool makes sense.